### PR TITLE
[MU4] Fix build, and warnings too

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
Needs some input from @andreituicu (as this is related to #6343)

### CODE REVIEW

File: mu4/importexport/internal/braille/exportbraille.cpp [hunk 1]
```diff
@@ -232,9 +232,9 @@
 #define BRAILLE_INVERTED_TURN_NOTE      QString(",4l")
 
 // Table 16.C. Page 13. Music Braille Code 2015
-#define BRAILLE_MORDENT                 QString("\"6")
+#define BRAILLE_SHORT_TRILL             QString("\"6")
 #define BRAILLE_TREBLEMENT              QString(";6")
-#define BRAILLE_INVERTED_MORDENT        QString("\"6l")
+#define BRAILLE_MORDENT                 QString("\"6l")
```
Comment (Jojo-Schmitz) [inline mu4/importexport/internal/braille/exportbraille.cpp:237]: these switched in SMuFL 1.3

File: mu4/importexport/internal/braille/exportbraille.cpp [hunk 2]
```diff
@@ -515,7 +515,7 @@ class ExportBraille
     QString brailleArticulation(Articulation* articulation);
     QString brailleFingeringAfter(Fingering* fingering);
     QString brailleGlissando(Note* note);
-    QString brailleRepeatMeasure(RepeatMeasure* repeatMeasure);
+    //QString brailleRepeatMeasure(RepeatMeasure* repeatMeasure); // ToDo?!
```
Comment (Jojo-Schmitz) [inline mu4/importexport/internal/braille/exportbraille.cpp:518]: @IsaacWeiss ?

File: mu4/importexport/internal/braille/exportbraille.cpp [hunk 3]
```diff
@@ -1100,8 +1100,8 @@ QString ExportBraille::brailleMeasure(Measure* measure, int staffCount)
             out << brailleBreath(toBreath(el));
         } else if (el->isBarLine() && toBarLine(el) != lastBarline(measure, el->track())) {
             out << brailleBarline(toBarLine(el));
-        } else if (el->isRepeatMeasure()) {
-            out << brailleRepeatMeasure(toRepeatMeasure(el));
+        //} else if (el->isRepeatMeasure()) { // ToDo ?!
+        //    out << brailleRepeatMeasure(toRepeatMeasure(el));
```
Comment (Jojo-Schmitz) [inline mu4/importexport/internal/braille/exportbraille.cpp:1104]: @IsaacWeiss ?

File: mu4/importexport/internal/braille/exportbraille.cpp [hunk 4]
```diff
@@ -1683,6 +1683,11 @@ QString ExportBraille::brailleBarline(BarLine* barline)
     case BarLineType::END_START_REPEAT:
         barlineBraille = BRAILLE_BARLINE_START_REPEAT + " " + BRAILLE_BARLINE_END_REPEAT;
         break;
+    case BarLineType::REVERSE_END:
+    //case BarLineType::REVERSE_FINALE: // duplicate of the above, alternative name
+    case BarLineType::HEAVY:
+    case BarLineType::DOUBLE_HEAVY:
+         break; // ToDo ?!
```
Comment (Jojo-Schmitz) [inline mu4/importexport/internal/braille/exportbraille.cpp:1690]: @andreituicu how to deal with those? They got added to MuseScore recently

File: mu4/importexport/internal/braille/exportbraille.cpp [hunk 5]
```diff
@@ -2050,8 +2055,8 @@ QString ExportBraille::brailleArticulation(Articulation* articulation)
         return BRAILLE_TRILL;
     case SymId::ornamentMordent:
         return BRAILLE_MORDENT;
-    case SymId::ornamentMordentInverted:
-        return BRAILLE_INVERTED_MORDENT;
+    case SymId::ornamentShortTrill:
+        return BRAILLE_SHORT_TRILL;
```
Comment (Jojo-Schmitz) [inline mu4/importexport/internal/braille/exportbraille.cpp:2059]: as said above, switched in SMuFL 1.3

File: mu4/importexport/internal/braille/exportbraille.cpp [hunk 6]
```diff
@@ -2127,6 +2132,7 @@ QString ExportBraille::brailleGlissando(Note* note)
     return QString();
 }
 
+#if 0 // ToDo ?!
```
Comment (Jojo-Schmitz) [inline mu4/importexport/internal/braille/exportbraille.cpp:2135]: @IsaacWeiss ?

File: mu4/importexport/internal/braille/exportbraille.cpp [hunk 7]
```diff
@@ -2481,6 +2488,7 @@ QString ExportBraille::brailleMarker(Marker* marker)
     case Marker::Type::FINE:
         return BRAILLE_FINE;
     case Marker::Type::TOCODA:
+    case Marker::Type::TOCODASYM:
```
Comment (Jojo-Schmitz) [inline mu4/importexport/internal/braille/exportbraille.cpp:2491]: added recently, I guess it is OK to treat it the same as the 'normel' to COda, @andreituicu

**Comment by igorkorsukov** [comment]
needs fix code style

**Comment by Jojo-Schmitz** [comment]
done

**Comment by igorkorsukov** [comment]
@Jojo-Schmitz needs fix mtest (or turn off it)
<diff -u /home/runner/work/MuseScore/MuseScore/mtest/braille/testOrnaments_Example_16.5_MBC2015_ref.brf testOrnaments_Example_16.5_MBC2015.brf failed

vtests expectedly has broken

**Comment by Jojo-Schmitz** [comment]
OK, mtests passed, vtests not

File: mu4/importexport/internal/braille/exportbraille.cpp (hunk 4 above)
Comment (Jojo-Schmitz) [reply mu4/importexport/internal/braille/exportbraille.cpp]: Actually I don't think we should be ignoring them, at least report them as normal barlines?

**Pull request merged**